tests/test_data/override_severity/test.robot

```diff
@@ -1,6 +1,6 @@
 *** Settings ***
 Documentation    Testing Robocop
-Force Tags       comeANDgo    # Evoke rule 0602 with severity I
+Default Tags       comeANDgo    # Evoke rule 0602 with severity I
```

Comment:
It invokes different rule now (Force Tags is deprecated in 6+) that's why I used Default Tags to trigger 0602